subsys/net/ip/net_pkt.c

```diff
@@ -315,10 +315,10 @@ void net_pkt_print_frags(struct net_pkt *pkt)
 	while (frag) {
 		total += frag->len;
 
-		frag_size = frag->size;
```

Comment:
That's also a valuable commit that could go on its own PR I believe

Reply:
Lets keep this commit here for time being. I can move it later if this PR gets delayed.

Reply:
I sent a separate PR for this fix #72148